public/docs/ts/latest/guide/i18n.jade

```diff
@@ -86,12 +86,12 @@ a(id="angular2-i18n")
   Whatever approach we decide to use, we must update our templates to define which strings will be translated.
  
   
-a(id="i18n-directive")
+a(id="i18n-directives")
 .l-main-section
 :marked
-  ### i18n directive
+  ### i18n directives
```

Comment:
It's actually not directives (notice that later in the review) - may be "markers" ?